src/components/slider/slider.st.css

```diff
@@ -0,0 +1,309 @@
+}
+
+.root:y-reverse {
+    height: 100%;
+}
+
+.slider {
+    cursor: pointer;
+    outline: none;
+    user-select: none;
+    pointer-events: inherit;
+    touch-action: none;
+    -webkit-tap-highlight-color: rgba(0,0,0,0);
+    -webkit-touch-callout: none;
+}
+
+.root:x .slider{
+    width: 100%;
+    padding: 15px 0;
+}
+
+.root:x-reverse .slider{
+    width: 100%;
+    padding: 15px 0;
+}
+
+.root:y .slider{
+    min-height: 36px;
+    height: 100%;
+    padding: 0 15px;
+}
+
+.root:y-reverse .slider{
+    min-height: 36px;
+    height: 100%;
+    padding: 0 15px;
+}
+
+.track {
+    position: relative;
+    border-radius: 2px;
+    background-color: #adbbc6;
+}
+
+.root:x .track{
+    height: 4px;
+    width: 100%;
+}
+
+.root:x-reverse .track{
+    height: 4px;
+    width: 100%;
+}
+
+.root:y .track{
+    text-align: left;
+    width: 4px;
+    height: 100%;
+    min-height: 36px;
+}
+
+.root:y-reverse .track{
+    text-align: left;
+    width: 4px;
+    height: 100%;
+    min-height: 36px;
+}
+
+.progress {
+    position: absolute;
+    background-color: #4a90e2;
+    border-radius: 2px;
+}
+
+.root:x .progress{
+    left: 0;
+    height: inherit;
+}
+
+.root:x-reverse .progress{
+    right: 0;
+    height: inherit;
+}
+
+.root:y .progress{
+    bottom: 0;
+    width: inherit;
+}
+
+.root:y-reverse .progress{
+    top: 0;
+    width: inherit;
+}
+
+.handle {
+    position: absolute;
+    width: 16px;
+    height: 16px;
+    border-radius: 8px;
+    background-color: #ffffff;
+    border: solid 1px #7d8c98;
+    box-sizing: border-box;
+    z-index: 1000;
+}
+
+.root:x .handle{
+    margin: -6px 0 0 -8px;
+}
+
+.root:x-reverse .handle{
+    margin: -6px -8px 0px 0px;
+}
+
+.root:y .handle{
+    margin: 0px 0px -8px -6px;
+}
+
+.root:y-reverse .handle{
+    margin: -8px 0px 0px -6px;
+}
+
+.markProgress,
+.markTrack
+{
+    width: 8px;
+    height: 8px;
+    position: absolute;
+    border-radius: 4px;
+}
+.markProgress{
+    background-color: #4a90e2;
+}
+.markTrack{
+    background-color: #adbbc6;
+}
+
+
+.root:x .markProgress,
+.root:x .markTrack
+{
+    margin: -2px 0 0 -4px;
+}
+
+.root:x-reverse .markProgress,
+.root:x-reverse .markTrack
+{
+    margin: -2px -4px 0 0;
+}
+
+.root:y .markProgress,
+.root:y .markTrack
+{
+    margin: 0 0 -4px -2px;
+}
+
+.root:y-reverse .markProgress,
+.root:y-reverse .markTrack
+{
+    margin: -4px 0 0 -2px;
+}
+
+.tooltip{
+    position: absolute;
+    text-align: center;
+    left: calc(50%);
+    bottom: 20px;
+}
+.tooltip > *{
+    display: inline-block;
+    position: relative;
+    left: calc(-50%);
+}
+
+/* Hover state */
+.slider:hover .track {
+    background-color: #7d8c98;
+}
+
+.slider:hover .markTrack {
+    background-color: #7d8c98;
+}
+
+.slider:hover .handle {
+    background-color: #edf7ff;
+    border: solid 1px #4a90e2;
+}
+
+/* Focus state */
+.handle:focus {
+    -webkit-box-shadow: 0 0 0 4px rgba(92, 180, 255, 0.5);
+    box-shadow: 0 0 0 4px rgba(92, 180, 255, 0.5);
```

Comment:
not sure about this one, but most likely we have predefined shadows in styleguide. 
@oleksiik-uxd can you help with this?

Reply:
focus has same style in all components, 
solid 4px var(--PR-3-50);